test/kpod.bats

```diff
@@ -47,3 +51,39 @@ KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
 	echo "$output"
 	[ "$status" -ne 0 ]
 }
+
+@test "kpod inspect image" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS pull redis:alpine
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} $KPOD_OPTIONS inspect redis:alpine
+    echo "$output"
+    [ "$status" -eq 0 ]
+}
+    run ${KPOD_BINARY} $KPOD_OPTIONS rmi redis:alpine
+
+@test "kpod inspect non-existed container" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS inspect 14rcole/non-existent
+    echo "$output"
+    [ "$status" -ne 0 ]
+}
+
+@test "kpod inspect with format" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS pull redis:alpine
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} $KPOD_OPTIONS --format {{.ID}} inspect redis:alpine
+    [ "$status" -eq 0]
+    inspectOutput="$output"
+    run ${KPOD_BINARY} $KPOD_OPTIONS images --quiet redis:alpine
+    [ "$status" -eq 0]
+    [ "$output" -eq "$inspectOutput" ]
+    run ${KPOD_BINARY} $KPOD_OPTIONS rmi redis:alpine
+}
+
+@test "kpod insepct specified type" {
+    run ${KPOD_BINARY} $KPOD_OPTIONS pull redis:alpine
+    [ "$status" -eq 0 ]
+    run ${KPOD_BINARY} $KPOD_OPTIONS inspect --type image redis:alpine
```

Comment:
The tests should check the output is correctly formatted by piping it into a validator.